fidesctl/src/fidesapi/main.py

```diff
@@ -33,6 +34,20 @@ async def health() -> Dict:
     return {"data": {"message": "Fides service is healthy!"}}
 
 
+@app.post("/admin/db/{action}", tags=["Admin"])
+async def db_action(action: str, database_url: str) -> Dict:
```

Comment:
It would be slightly more secure if the `database_url` were accepted as part of the `POST` request body, not a query param. Looking at the FastAPI docs, this can be done by requiring that it's passed to this function as a custom class type, but it wasn't obvious where that class definition should live such that it can be imported everywhere it would be needed.

Reply:
I'm assuming this will come up again, so maybe we should just create a new module in `fidesapi` for models we want to reuse?

Reply:
wait, sorry rereading this the CLI doesn't need to send the database url, the API can get it from the config like it does when it inits the database as well

Reply:
the idea is that people can deploy the cli with no knowledge of the database, including the URL

Reply:
I haven't used an interpreted language in a while 😅